city_scrapers/spiders/chi_ssa_20.py

```diff
@@ -0,0 +1,105 @@
+    timezone = "America/Chicago"
+    start_urls = ["https://www.mpbhba.org/business-resources/"]
+    location = {
+        "name": "Beverly Bank & Trust,",
+        "address": "10258 s. Western ave.",
+    }
+
+    def parse(self, response):
+
+        base = response.xpath(
+            "//*[self::p or self::strong or self::h3]/text()"
+        ).getall()
+
+        base = [re.sub(r"\s+", " ", item).lower() for item in base]
+
+        for index, line in enumerate(base):
+            if "ssa meetings" in line:
+                del base[:index]
+
+        for index, line in enumerate(base):
+            if "ssa 64" in line:
+                del base[index:]
+
+        for item in base:
+            if re.match(r"^\D*\d{4}\D*$", item):
+                year = re.match(r"^\d{4}", item)[0]
+
+        for item in base:
+
+            # don't pass empty lines to methods
+            if re.match(r"^\s*$", item):
+                continue
+
+            start = self._parse_start(item, year)
+            if not start:
+                continue
+
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description=self._parse_description(item),
+                classification=self._parse_classification(item),
+                start=start,
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        return "SSA 20"
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item, year):
+
+        if not any(word in item for word in ["beverly", "ssa"]):
+            item = re.sub(r"([,\.])", "", item).strip()
+            ready_date = item + " " + year
+            date_object = datetime.strptime(ready_date, "%A %B %d %I %p %Y")
+            return date_object
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
+        return ""
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_links(self, item):
+        """Parse or generate links."""
+        return [{"href": "", "title": ""}]
```

Comment:
We should try and parse these from the section on the right and attempt to match them to meetings. You can see an example of this in [`chi_il_medical_district`](https://github.com/City-Bureau/city-scrapers/blob/c42620480f9184ca74885e2a3c6ef576faba3eb7/city_scrapers/spiders/chi_il_medical_district.py)

Reply:
I will try to accomplish this. I'm afraid I may have chosen a less than optimal logic if including the links should be involved,
(We had a small conversation about this on Slack, irrc). 

I will try and see how this option can be factored in.

Reply:
Another item that's worth mentioning: The links on the right do not have dates posted on them but rather "1st quarter 2018" , "1st quarter 2019" and so on. 
The meetings posted for 2019 ("current" at the time, I assume) were in June and July - not necessarily corresponding to a quarterly schedule, that could be a challenge, too.

Reply:
As they've added 2021 meetings, things changed up a bit. Before the change, all the meetings took place in recent history, with some of them - having matching links on the right. 

Now, there are only future meetings on the left (2021), and everything on the right is past, with no matching meetings on the left (future). 
So it's a new challenge, so it seems.

Reply:
@pjsier With the added 2021 meetings:

A meeting is either current/upcoming _without_ meeting minutes, but _with_ time data or, it's past _with_ meeting minutes and _without_ time data. 
What should we do in this case ?
https://www.mpbhba.org/business-resources/

An example to a past meeting link:
```
 <div class="et_pb_text_inner"><p>SSA 20 1st Quarter 2020 meeting minutes</p></div>
```
Which does not provide any time data, more like a link to archive.

Reply:
In general if the time data isn't present on past meetings we can go with a reasonable default based on the current meetings. It looks like 8am should work?

Reply:
8am works for me. What about the date  :) ?

Reply:
Ah missed that, for some of them it looks like the date is in the URL itself, and for the others we might just need to ignore them for now